docker-dev/docker-compose.override.yml

```diff
@@ -0,0 +1,76 @@
+# Docker compose override file to map the code directories inside the 
+# container, allowing the app to rebuild dynamically
+
+# To start the docker dev environment, run this command from the root folder : 
+# docker compose -f docker-compose.yml -f docker-dev/docker-compose.override.yml up
+
+services:
+  transition-www:
+    command:
+      - /bin/sh 
+      - /app/docker-dev/entrypoint-dev.sh
+    volumes:
+
+      # Bind mounts on the package directories. These enable instant code changes on the container without entirely rebuilding the image    
```

Comment:
Can you add the comment here about why we need to add all those file. So that we know if we add modules or other things what we need to add here.